docs/src/increase-observability/integrate-observability/blackfire.md

```diff
@@ -34,10 +34,13 @@ All customers can also subscribe to Blackfire separately.
 
 ### On a Grid plan or {{% names/dedicated-gen-3 %}} infrastructure 
 
-If you have the Platform.sh [Observability Suite](https://platform.sh/features/observability-suite/), 
-the [Blackfire automated integration](#automated-integration) is enabled on Grid environments by default. 
+If you have subscribed to a Grid plan complete with the Platform.sh [Observability Suite](https://platform.sh/features/observability-suite/),
+the [Blackfire automated integration](#automated-integration) is enabled on your environments by default. 
 
-Otherwise, to enable Blackfire on your PHP or Python applications,
+If you have subscribed to a Grid plan without the Observability suite,
+or if you are a {{% names/dedicated-gen-3 %}} user,
+the Blackfire automated integration isn't enabled on your environments.
+To enable Blackfire on your PHP or Python apps,
```

Comment:
:hammer_and_wrench: To shorten things a little and focus on what to do, rather than what doesn't happen.

```suggestion
If you have Grid environments without the Observability Suite or {{% names/dedicated-gen-3 %}} environments,
you need to enable the integration yourself.
To enable Blackfire on your PHP or Python apps,
```